### DISCUSSION TITLE
如何将blog和docs共存？ How can I use blog and docs in the same time?

### DISCUSSION THREAD

**Discussion by AstralFlare-owo** [vuepress-theme-hope/vuepress-theme-hope#4421] [Q&A] [answered]
`layout: BlogHome` 会导致前端页面停止工作。

`layout: BlogHome` makes the frontend page stop responding.

**Comment by Mister-Hope** [comment, marked as answer]
That's because you DO NOT enable blog feature at all. Set `plugins.blog: true` in theme options first.

**Reply by AstralFlare-owo** [reply]
thanks

**Reply by AstralFlare-owo** [reply]
> thanks

I'm looking for that. Thanks for your help.
